### PR TITLE
Adding python script for adding users

### PR DESCRIPTION
**Pull request by warrenbailey**
Helper scripts for add users and groups to UAA

### CODE REVIEW

File: scripts/add_user_to_group.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    headers = {'Content-Type': 'application/x-www-form-urlencoded',
+               'Accept': 'application/json'}
+    payload = {'grant_type': 'client_credentials',
+               'response_type': 'token',
+               'token_format': 'opaque'}
+    response = requests.post(f'http://{url}/oauth/token', headers=headers,
+                             params=payload,
+                             auth=(client_id, client_secret))
+    resp_json = response.json()
+    pprint.pprint(resp_json)
```
Comment (JamesGardiner) [inline scripts/add_user_to_group.py]: Do we want to print all this?

File: scripts/add_user_to_group.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import argparse
```
Comment (JamesGardiner) [inline scripts/add_user_to_group.py]: If this is being run as a script it will be worth adding a shebang

File: scripts/add_user_to_group.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+parser.add_argument("-g", "--group", required=True, dest="group", help="The group name")
+parser.add_argument("-u", "--username", required=True, dest="username", help="The username to add to the group")
+
+
+args = parser.parse_args()
+
+token = login_client(client_id=args.client_id, client_secret=args.client_secret, url=args.url)
+group_id = query_group(access_token=token, group=args.group, url=args.url)
+user_id = query_user(access_token=token, username=args.username, url=args.url)
+add_user_to_group(access_token=token, user_id=user_id, group_id=group_id, url=args.url)
```
Comment (JamesGardiner) [inline scripts/add_user_to_group.py]: Might want to add a if `__name__ == '__main__'` as well

File: scripts/create_client.py [hunk 4]
```diff
@@ -1,10 +1,12 @@
 import argparse
 import json
-
+import pprint
 import requests
```
Comment (JamesGardiner) [inline scripts/create_client.py]: Gonna have to ask you to make these imports PEP8 compliant 👍

File: scripts/create_client.py [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
 
 
 def login_admin(admin_id, admin_secret, url):
+    print(admin_id)
```
Comment (JamesGardiner) [inline scripts/create_client.py]: Do we really want to print these?

File: scripts/add_user_to_group.py (hunk 1 above)
Comment (warrenbailey) [reply scripts/add_user_to_group.py]: It's useful for debugging but I'll hide it behind a verbose flag

File: scripts/add_user_to_group.py (hunk 2 above)
Comment (warrenbailey) [reply scripts/add_user_to_group.py]: agreed

File: scripts/add_user_to_group.py (hunk 3 above)
Comment (warrenbailey) [reply scripts/add_user_to_group.py]: Will do

**Pull request merged**